modin/core/io/file_dispatcher.py

```diff
@@ -338,3 +338,25 @@ def build_partition(cls, partition_ids, row_lengths, column_widths):
                 for i in range(len(partition_ids))
             ]
         )
+
+    @classmethod
+    def _file_not_found_msg(cls, filename: str):
+        """
+        Produce a message indicating that a file was not found.
+
+        Parameters
+        ----------
+        filename : str
+            The name of the file that was not found.
+
+        Returns
+        -------
+        str
+            A message indicating `filename` was not found.
+        """
+        return f"No file with name '{filename}'"
```

Comment:
IMHO it's easier for the reader if we omit this insanely large docstring for a one-line formatter, and just silence the doc_checker with correct `# noqa`-s